assume/common/base.py

```diff
@@ -139,25 +142,51 @@ def set_dispatch_plan(
         Iterates through the orderbook, adding the accepted volumes to the corresponding time slots
         in the dispatch plan. It then calculates the cashflow and the reward for the bidding strategies.
 
+        Additionally, updates the average operation and downtime dynamically.
```

Comment:
As this is something very different, I would like to have this part in a separate function, to maintain the right scope.

The downtime was removed - so should be removed from comment as well

Probably it would also be better to call the function then in `execute_current_dispatch`, to not have the same problem with multi market bidding as well..?

Reply:
@maurerle removed it from base into calculate_reward, which is fitting